modules/tinymce/changelog.txt

```diff
@@ -16,6 +16,7 @@ Version 5.4.0 (TBD)
     Changed stylesheet loading, so that UI skin stylesheets now to load in a ShadowRoot, if the editor is in a ShadowRoot. #TINY-6089
     Changed the DOM location of menus so that they display correctly when the editor is in a ShadowRoot. #TINY-6093
     Changed the table plugin to correctly detect all valid header row structures #TINY-6007
+    Changed custom element handling to support a broader set of names #5971
```

Comment:
This shouldn't be `Changed`, it should either be `Added` or `Fixed`. Given we already support custom elements, this should probably be `Fixed`. Additionally, this is a little too broad of a statement as this is just fixing the ability to set the valid children for custom elements.